Cargo.toml

```diff
@@ -8,16 +8,16 @@
 	name = "winconsole"
 	readme = "README.md"
 	repository = "https://github.com/omarkmu/winconsole"
-	version = "0.10.0"
+	version = "0.11.0"
```

Comment:
I guess I can directly do a review, are there breaking changes? I don't see them in this PR, so why not doing a minor version upgrade like 0.10.1?

Reply:
True, I changed the version.